### ISSUE TITLE
Docker version error

### ISSUE THREAD

**Issue by AntZee75** [SensorsIot/IOTstack#503]
Hi!

I'm new in this world. I've tried to install the IOTstack, but I've got the following error messages:

```
pi@raspberrypi:~/IOTstack $ python --version
Python 3.9.2
pi@raspberrypi:~/IOTstack $ docker --version
Docker version 20.10.5+dfsg1, build 55c4c88
pi@raspberrypi:~/IOTstack $ ./menu.sh
Checking for project update
From https://github.com/SensorsIot/IOTstack
 * branch            master     -> FETCH_HEAD
Project is up to date
Python Version: 'Python 3.9.2'. Python is up to date.
ruamel.yaml Version: 'ruamel.yaml 0.16.12'. ruamel.yaml is up to date.
Blessed Version: 'blessed 1.19.1'. Blessed is up to date.
Please enter sudo pasword if prompted
Command: docker version -f "{{.Server.Version}}"
Error getting docker version. Received permission denied error. Try running with: ./menu.sh --run-env-setup
pi@raspberrypi:~/IOTstack $ ./menu.sh --run-env-setup
Checking for project update
From https://github.com/SensorsIot/IOTstack
 * branch            master     -> FETCH_HEAD
Project is up to date
Python Version: 'Python 3.9.2'. Python is up to date.
ruamel.yaml Version: 'ruamel.yaml 0.16.12'. ruamel.yaml is up to date.
Blessed Version: 'blessed 1.19.1'. Blessed is up to date.
Please enter sudo pasword if prompted
Command: docker version -f "{{.Server.Version}}"
Error getting docker version. Received permission denied error. Try running with: ./menu.sh --run-env-setup
pi@raspberrypi:~/IOTstack $
pi@raspberrypi:~/IOTstack $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
pi@raspberrypi:~/IOTstack $

```

Brand new Raspbian install, all is up to date. What can I do with this error?

**Comment by ukkopahis** [comment]
Try running:
`docker run hello-world`

Solution might be:
https://stackoverflow.com/questions/48957195/how-to-fix-docker-got-permission-denied-issue

**Comment by AntZee75** [comment]
Thanks!

It's worked. After a reboot, the problem goes to like this: https://github.com/SensorsIot/IOTstack/issues/496.
But this helped me! Thanks again.

**Comment by ukkopahis** [comment]
And that issue is fixed in pull-request #498 

As a convenience that and many other yet-to-be merged changes are available in my fork's master-branch:
https://github.com/ukkopahis/IOTstack/
